references/backup.md

```diff
@@ -0,0 +1,58 @@
+# Backup
+
+The `backup` route allow user to trigger a backup creation and download created backup.
+
+## Trigger backup
+
+<RouteHighlighter method="POST" route="/backup"/>
+
+Trigger a backup creation process
+
+### Example
+
+<code-samples id="post_backup_1" />
+
+#### Response: `202 Accepted`
+
+```json
+{
+    "id": 5
+}
+```
+
+## Get backup status
+
+<RouteHighlighter method="GET" route="/backup/:backup_uid/status"/>
+
+Get backup status.   
+Returned status could be:
+
+- `processing`: backup creation is not finished
+- `backup_process_failed`: an error occured during backup process, task aborted
+- `done`: backup creation is finished
```

Comment:
```suggestion
- `processing`: Backup creation is not finished.
- `backup_process_failed`: An error occured during backup process, task aborted.
- `done`: Backup creation is finished.
```

Side question: `backup_process_failed` this status can only appear when creating a backup or also when downloading a backup?